assets/js/blocks/mini-cart/quantity-badge/index.tsx

```diff
@@ -8,20 +8,20 @@ import { Icon } from '@wordpress/icons';
  * Internal dependencies
  */
 import './style.scss';
-import { IconType } from '.././types';
+import { IconType, ColorItem } from '.././types';
 
 interface Props {
 	count: number;
 	icon?: IconType;
-	iconColor?: string;
-	productCountColor?: string;
+	iconColor: ColorItem | { color: undefined };
+	productCountColor: ColorItem | { color: undefined };
```

Comment:
This is where inconsistency of types needs cleaned up. You'll notice that through these changes.